core/src/main/java/org/jsmart/zerocode/core/engine/mocker/RestEndPointMocker.java

```diff
@@ -25,10 +28,23 @@
 
     public static WireMockServer wireMockServer;
 
+    public static Boolean buildUrlPathEqualToPattern_FeatureFlag = true;
+
+    private static boolean hasMoreThanOneStubForSameUrl(List<String> urls) {
+        if(urls == null) return false;
```

Comment:
url list won't be empty(otherwise what are we mocking?) , so this check is redundant, you can remove